CHANGELOG.md

```diff
@@ -22,6 +22,8 @@
 
 * **[Feature]** Support sending attachments in handled errors.
 
+* **[Fix]** Fix an error on running same application twice and enabling App Center.
```

Comment:
This needs to be a new version 2.6.2

Reply:
Please catch up with develop: https://github.com/microsoft/appcenter-sdk-dotnet/pull/1210/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1e

Reply:
Fixed